### PR TITLE
Add xorg-libxxf86vm

### PR DESCRIPTION
**Pull request by zklaus**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by hmaarrfk** [comment]
Is this really necessary on windows? Are you trying to build a package for windows that requires this?

Otherwise, maybe skip and you can use on linux!

**Comment by zklaus** [comment]
Thanks for the comment. It's a good question! After starting this package, I saw that at the moment [libxxf86vm is one of the few CDT packages](https://conda-forge.org/docs/maintainer/knowledge_base.html#cdt-packages) and now I am wondering how wise it is to package it in this way even for linux. On the other hand, it is not unprecedented. Do you have any insight into this?

**Comment by hmaarrfk** [comment]
I think there has been a push to get more and more of x11 into conda-forge.

MIght as well just disable Windows Build and ask for a final review.

File: recipes/xorg-libxxf86vm/build.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    --disable-silent-rules
+)
+
+./configure "${configure_args[@]}"
+make -j$CPU_COUNT
+make install
+make check
+rm -rf $uprefix/share/man $uprefix/share/doc/${PKG_NAME#xorg-}
+
+# Remove any new Libtool files we may have installed. It is intended that
```
Comment (hmaarrfk) [inline recipes/xorg-libxxf86vm/build.sh:63]: I think this is now done automatically

**Pull request closed without merging**